spesmilo/electrum
# Convert2cc

* add new plugin convert2cc

Testing:
1. create multisig/singlesig wallet with non coldcard hardware device/s (in my case trezor, ledger)
2. click on 'Wallet' tab and choose 'Information' - wallet information dialog will open and you should see 'convert2cc' button ![Screenshot from 2022-03-07 09-04-40](https://user-images.githubusercontent.com/25349625/157004878-c592cb6f-e8dc-4586-bc95-0eaf12ce3191.png)
3. if 2. does not hold - go to plugins, activate 'convert2cc' and go back to 2. (needs ckcc-protocol as dependency)
4. if wallet created in 1. is singlesig disconnect hww and load same seed + passphrase (if used) to coldcard and connect it to PC, if multisig, do the same for just one cosigner
   (can be done even without Coldcard connected, it won't give you a hint based on master fingerprint match)
5. click 'convert2cc' button in wallet information modal - convert2cc modal will pop up 
![Screenshot from 2022-03-07 09-05-07](https://user-images.githubusercontent.com/25349625/157005052-08a1c926-7dae-4cd2-904c-3d7ce7bec423.png)
6. one of the keystores/cosigner should match with your connected coldcard (icon_paired + [matches connected Coldcard])
   if your coldcard is not connected there will be no match ![Screenshot from 2022-03-07 09-40-07](https://user-images.githubusercontent.com/25349625/157005186-8d048dba-91ef-46d1-a83e-758ec74f8613.png)
8. choose correct keystore and hit 'convert button'
9. select how to name a where to store newly created converted wallet
10. hit yes - confirming that you want to open newly created wallet
11. old wallet is closed, converted wallet is opened and one of your devices is converted to Coldcard (should show paired icon)
12. try that wallet is working as expected
